experimental/packages/opentelemetry-instrumentation-grpc/README.md

```diff
@@ -54,6 +54,42 @@ gRPC instrumentation accepts the following configuration:
 | [`metadataToSpanAttributes`](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation-grpc/src/types.ts#L27) | `object`          | List of case insensitive metadata to convert to span attributes. Client and server (outgoing requests, incoming responses) metadata attributes will be converted to span attributes in the form of `rpc.{request\response}.metadata.metadata_key`, e.g. `rpc.response.metadata.date` |
 
+## Semantic Conventions
+
+Up to and including v0.200.0, `instrumentation-grpc` generates telemetry using [Semantic Conventions v1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md).
+
+HTTP semantic conventions (semconv) were stabilized in v1.23.0, and a [migration process](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/http-migration.md#http-semantic-convention-stability-migration) was defined.
+`instrumentation-grpc` versions 0.201.0 and later include support for migrating to stable HTTP semantic conventions, as described below.
+The intent is to provide an approximate 6 month time window for users of this instrumentation to migrate to the new HTTP semconv, after which a new minor version will use the *new* semconv by default and drop support for the old semconv.
+See the [HTTP semconv migration plan for OpenTelemetry JS instrumentations](https://github.com/open-telemetry/opentelemetry-js/issues/5646).
+
+To select which semconv version(s) is emitted from this instrumentation, use the `OTEL_SEMCONV_STABILITY_OPT_IN` configuration option.
+
+- `http`: emit the new (stable) v1.23.0 semantics
+- `http/dep`: emit **both** the old v1.7.0 and the new (stable) v1.23.0 semantics
+- By default, if `OTEL_SEMCONV_STABILITY_OPT_IN` includes neither of the above tokens, the old v1.7.0 semconv is used.
+
+**Span attributes:**
+
+| v1.7.0 semconv  | v1.23.0 semconv  | Notes                                                                                                         |
+| --------------- | ---------------- | ------------------------------------------------------------------------------------------------------------- |
+| `net.peer.name` | `server.address` | Server domain name if available without reverse DNS lookup; otherwise, IP address or Unix domain socket name. |
+| `net.peer.port` | `server.port`    | Server port number                                                                                            |
+| `rpc.method`    |                  | The name of the (logical) method being called, must be equal to the $method part in the span name.            |
+| `rpc.service`   |                  | The full (logical) name of the service being called, including its package name, if applicable.               |
+| `rpc.system`    |                  | A string identifying the remoting system.                                                                     |
```

Comment:
This can be read as saying that the `rpc.*` ones are going away when the newer HTTP semconv is selected. When I formatted the table like this for instr-fetch it worked because *all* its attributes are HTTP-related.

How about something like:

```
**Span attribute changes:**

... table with just server.address and server.port ...

**Other span attributes:**

| Name | Notes |
| ---- | ----- |
| `rpc.method` | ... |
...

(and could include `rpc.grpc.status_code` too)
```

Or possibly not bother with the `Other span attributes` table.